tools/sgen/input/FluentUIStyle.yml

```diff
@@ -420,17 +421,11 @@ MSFGhostButtonTokens extends MSFButtonTokens:
   borderColor: [ rest: $Colors.Neutral.clear, hover: $Colors.Neutral.clear, pressed: $Colors.Neutral.clear, selected: $Colors.Brand.shade10, disabled: $Colors.Neutral.clear ]
 
 AP_MSFListTokens:
-  backgroundColor: $Colors.Background.neutral1
   borderColor: $Colors.Stroke.neutral1
-  borderSize: $Border.size.thin
-  horizontalCellPadding: $Spacing.medium
-  sublabelColor: $Colors.Foreground.neutral3
-  sublabelFont: $Typography.caption1
 
 AP_MSFListCellTokens:
   backgroundColor: [ rest: $Colors.Background.neutral1, pressed: $Colors.Background.neutral5 ]
   borderColor: $Colors.Stroke.neutral1
```

Comment:
This borderColor is not being used.
Is this the color that gets applied to the cell divider? (ListCell.swift, line 114)
If this is indeed not used, let's delete it from here, generated files and the ListTokens.swift (line 62)!

Reply:
Thanks for the offline conversation on the "Divider x Rectangle" issue! 

I think we're good to go with one of them based on what the design team decides with regards to the thickness (Rectangle if they want full control of the thickness versus Divider if they just want to have the default value from the platform).

I believe this is the last standing issue on the PR before we ca merge it.